python/cudf/cudf/core/column/struct.py

```diff
@@ -190,3 +192,35 @@ def field(self, key):
             return self._return_or_inplace(self._column.children[pos])
         else:
             return self._return_or_inplace(self._column.children[key])
+
+    def explode(self):
+        """
+        Return a DataFrame whose columns are the fields of this struct Series.
+        Note that a copy of the columns is made.
```

Comment:
Maybe put this in a [`Notes`](https://numpydoc.readthedocs.io/en/latest/format.html#notes) section? Or if you think it's a big enough issue for some users, a [`Warnings`](https://numpydoc.readthedocs.io/en/latest/format.html#warnings) section?

Reply:
I moved it to a `Notes` section -- ditto below